GOVERNANCE.md

```diff
@@ -0,0 +1,140 @@
+## Express Working Group Governance
+
+### Collaborators
+
+The nodejs/express GitHub repository is maintained by the WG and additional
```

Comment:
This document refers to the repository as `nodejs/express`, yet in other parts of the pull request reference the repository as `strongloop/express`. Can it consistently refer to the same one? Probably `strongloop/express` is the most appropriate for this pull request.